nodejs/awsx/ec2/vpc.ts

```diff
@@ -447,6 +409,31 @@ export interface VpcSubnetArgs {
     tags?: pulumi.Input<aws.Tags>;
 }
 
+/**
+ * Alias for a cidr block.
+ */
+export type CidrBlock = string;
+
+export interface VpcSubnetLocation {
+    /**
+     * The AZ for the subnet.
+     */
+    availabilityZone?: string;
+    /**
+     * The AZ ID of the subnet.
```

Comment:
Maybe comment that this is only used if `availabilityZone` is undefined?

Reply:
sure.